esp/src/eclwatch/templates/HPCCPlatformWidget.html

```diff
@@ -70,17 +72,23 @@
             </div>
         </div>
     </div>
-    <div id="${id}AboutDialog" title="About HPCC Platform" style="width: 480px;" data-dojo-type="dijit.Dialog">
+    <div id="${id}AboutDialog" title="${i18n.AboutHPCCSystems}" style="width: 480px;" data-dojo-type="dijit.Dialog">
```

Comment:
Should there be a space after "About," or is that just an internal tag ?

Reply:
internal tag